llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -3698,7 +3697,7 @@ void LoopVectorizationCostModel::collectLoopUniforms(ElementCount VF) {
 
     // Determine if all users of the induction variable are uniform after
     // vectorization.
-    auto UniformInd = llvm::all_of(Ind->users(), [&](User *U) -> bool {
+    bool UniformInd = llvm::all_of(Ind->users(), [&](User *U) -> bool {
```

Comment:
while you are editing the line, might as well drop redundant llvm::


```suggestion
    bool UniformInd = all_of(Ind->users(), [&](User *U) -> bool {
```